collector/interfaces.go

```diff
@@ -100,15 +100,16 @@ type EndPoint struct {
 // FlowRecord describes a flow record for statistis
 type FlowRecord struct {
 	ContextID        string
-	Count            int
 	Source           *EndPoint
 	Destination      *EndPoint
 	Tags             *policy.TagStore
-	Action           policy.ActionType
-	ObservedAction   policy.ActionType
 	DropReason       string
 	PolicyID         string
 	ObservedPolicyID string
+	Count            int
+	Action           policy.ActionType
+	ObservedAction   policy.ActionType
+	L4Proto          uint8
```

Comment:
Can we rename internal variables with complete names to make it clear ?
Use `L4Protocol` instead of `L4Proto`